### PR TITLE
Orderable crates: boxing equipment, empty boxes, bodybags

### PR DESCRIPTION
**Pull request by lapatison**
## About the PR

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Boxing equipment crates, empty boxes crates, and medical body bag crates now can be ordered from cargo

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Crates/fun.yml [hunk 1]
```diff
@@ -172,3 +172,18 @@
...
+- type: entity
+  id: CrateFunBoxing
+  parent: CrateGenericSteel
+  components:
+  - type: StorageFill
+    contents:
+      - id: ClothingHandsGlovesBoxingRed
+      - id: ClothingHandsGlovesBoxingBlue
+      - id: ClothingHandsGlovesBoxingYellow
+      - id: ClothingHandsGlovesBoxingGreen
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/Fills/Crates/fun.yml:185]: Aren't these combat items, I don't think they should be orderable.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Crates/fun.yml (hunk 1 above)
Comment (lapatison) [reply Resources/Prototypes/Catalog/Fills/Crates/fun.yml:185]: What do you mean by combat items? Like, yeah, they meant for boxing and have reduced brute and increased stam damage, but they're not OP or anything

**Comment by AJCM-git** [comment]
merge master

**Comment by lapatison** [comment]
> merge master

Done

**Comment by AJCM-git** [comment]
linter error

File: Resources/Prototypes/Catalog/Fills/Crates/fun.yml (hunk 1 above)
Comment (mirrorcult) [reply Resources/Prototypes/Catalog/Fills/Crates/fun.yml:185]: idk why these wouldnt be fine

**Review by mirrorcult — APPROVED** [review]

**Comment by mirrorcult** [comment]
linter still failing, doesnt look erroneous

File: Resources/Prototypes/Catalog/Fills/Crates/service.yml [hunk 2]
```diff
@@ -113,6 +113,14 @@
...
 - type: entity
+  id: CrateServiceBox
+  parent: CratePlastic
+  components:
+    - type: StorageFill
+      contents:
+        - id: BoxCardboard
+          amount: 6
+
   id: CrateJanitorBiosuit
```
Comment (lapatison) [inline Resources/Prototypes/Catalog/Fills/Crates/service.yml:124]: Gonna fix it today later

File: Resources/Prototypes/Catalog/Fills/Crates/fun.yml [hunk 3]
```diff
@@ -132,3 +132,18 @@
...
+      - id: UniformShortsRed
+        amount: 3
+      - id: UniformShortsRedWithTop
+        amount: 3
```
Comment (AJCM-git) [inline Resources/Prototypes/Catalog/Fills/Crates/fun.yml:149]: why 4 pairs of gloves but only 3 shorts

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Crates/fun.yml (hunk 3 above)
Comment (lapatison) [reply Resources/Prototypes/Catalog/Fills/Crates/fun.yml:149]: 6 shorts.
3 with tops, 3 without

**Review by AJCM-git — APPROVED** [review]

File: Resources/Prototypes/Catalog/Fills/Crates/fun.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Prototypes/Catalog/Fills/Crates/fun.yml:207]: In my brain I thought it was boxingcomp.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**